compare/compare.c

```diff
@@ -679,7 +691,8 @@ int main(int argc, char *argv[])
         if (!track)
             continue;
         Result res = process_image(renderer, track, list.items[i].dir, output,
-                                   name, list.items[i].time, scale);
+                                   name, list.items[i].time,
+                                   scale * anamorphic_scale, scale);
```

Comment:
This is up to &times;512 original bitmap size (previously only &times;64), I think it's overkill.

Reply:
TBH I don’t really see why we need to limit the scales at all. This is a utility that takes the scale explicitly. If the user wants a bigger one, who are we to stop them?

But sure, with `<scaleX:1-8>:<scaleY:1-8>` the limit will be smaller.

Reply:
`uint16_t` for intermediate during averaging can overflow. Also division approximation can break down too (actually I should check that).

Reply:
Hmm, does that mean it was broken for `-s 6`, too? Or is it something specific to the divisor of 35?

Reply:
No, for scales up to 8 the only problematic pair is 5&times;7. Why it's so specific I don't know, I've simply tested for all possible inputs.

Reply:
Committed in your name. If you prefer, I can squash it into the main `compare` commit.

Reply:
I think it's better to not keep regression in history.

Reply:
Squashed.